tests/formats/pdb.cpp

```diff
@@ -2,6 +2,9 @@
 // Copyright (C) Guillaume Fraux and contributors -- BSD license
 
 #include <fstream>
+
+#include <iostream>
```

Comment:
You should not need this input file

Reply:
Yeah, I was just using it for checking. I deleted this line in the last commit.

Reply:
Ok, sorry, I did not see your last commit 😄